homeassistant/components/hive/climate.py

```diff
@@ -42,6 +57,19 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     add_entities([climate])
 
+    @refresh_system
+    def heating_boost(service):
+        """Handle the service call."""
+        node_id = HiveSession.entity_lookup[service.data.get(ATTR_ENTITY_ID)]
```

Comment:
We don't know that the entity_id will be correct, right?
```py
node_id = HiveSession.entity_lookup.get(service.data[ATTR_ENTITY_ID])
if not node_id:
    # log or raise error
    return
```

Reply:
Done